tests/Oryx.Tests.Common/TestLanguageDetector.cs

```diff
@@ -7,36 +7,36 @@
 
 namespace Microsoft.Oryx.Tests.Common
 {
-    public class TestLanguageDetectorUsingLangName : ILanguageDetector
+    public class TestPlatformDetectorUsingLangName : IPlatformDetector
     {
         private readonly string _languageName;
         private readonly string _languageVersion;
 
-        public TestLanguageDetectorUsingLangName(string detectedLanguageName, string detectedLanguageVersion)
+        public TestPlatformDetectorUsingLangName(string detectedPlatformName, string detectedPlatformVersion)
         {
-            _languageName = detectedLanguageName;
-            _languageVersion = detectedLanguageVersion;
+            _languageName = detectedPlatformName;
+            _languageVersion = detectedPlatformVersion;
         }
```

Comment:
The naming of this file doesn't match with its class name within. Change its class name to `TestLanguageDetector` or change file name to `TestLanguageDetectorUsingLangName.cs`.

Reply:
I prefer to change the class name to `TestLanguageDetector`, but it looks like lots of places need to change.

Reply:
We chose to use 'platform' because the word 'language' does not suit everything...for example `python` is a language, but `dotnet` is not but C# is. Sure I will change the file names accordingly.